diagnostic.md

```diff
@@ -54,6 +66,21 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
+.big{
+  ...;
+}
+
+#contentPane{
+  ...;
+}
+
+.important .red{
```

Comment:
when you have a space, it is the descendent which will have the styling. To apply the styling to elements that only have BOTH the classes on it, it would be `.important.red`